releases/launch_3.0.0.yml

```diff
@@ -31,7 +32,7 @@
         keypair: AGR-ssl2
         group: ["default", "SSH", "ES Transport", "HTTP/HTTPS"]
         instance_type: "{{ WEBSERVER_INSTANCE_TYPE }}"
-        image: ami-0b0b90473c097c55a
+        image: "{{ WEBSERVER_AMI_IMAGE }}"
```

Comment:
Might be good to leave the AMI hard-coded in the production launch files, as changing the AMI can have far fetching consequences. Alternatively, if allowing instance types with different architectures is really needed for (old) production release files, you could define a default value for `WEBSERVER_AMI_IMAGE` in the production release file itself, which would then not change after creation of the file (unlike the value in [`environments/shared_aws_instance_variables.yml`](./environments/shared_aws_instance_variables.yml) which is more likely to change for future releases).
This comment applies for all production launch files.